### PR TITLE
Added Project

### PR DESCRIPTION
**Pull request by aaditya**
Added "sbrw-server" project.

### CODE REVIEW

File: _data/projects/nfsw-server.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- javascript
+- racing
+- backend
+- nodejs
+- nfsworld
+- sbrwserver
+- sbrw
+upforgrabs:
+  name: hobby project
+  link: https://github.com/aadityachakravarty/sbrw-server/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/nfsw-server.yml]: There aren't any issues listed on the repository for new contributors to look at.

Is that something you'd be able to setup before I merge this?

**Review by shiftkey — CHANGES_REQUESTED** [review]
@aadityachakravarty thanks for submitting! I have one question to start this off...

File: _data/projects/nfsw-server.yml (hunk 1 above)
Comment (aaditya) [reply _data/projects/nfsw-server.yml]: Added a few to begin with, will add more as the project goes on.

File: _data/projects/nfsw-server.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+tags:
+- javascript
+- racing
+- backend
+- nodejs
+- nfsworld
+- sbrwserver
+- sbrw
+upforgrabs:
+  name: hobby project
```
Comment (shiftkey) [inline _data/projects/nfsw-server.yml]: Issue list looks good - let's change this to be the label used on your project:

```suggestion
  name: up-for-grabs
```

**Comment by aaditya** [comment]
Committed the suggested name.

**Review by shiftkey — APPROVED** [review]
Thanks @aadityachakravarty!

**Pull request merged**